apps/web/pages/auth/login.tsx

```diff
@@ -152,7 +165,7 @@ export default function Login({
                   <PasswordField
                     id="password"
                     autoComplete="off"
-                    required
+                    required={!totpEmail}
```

Comment:
When a totp code is needed in an external provider login use case, there is no password involved, so not requiring it to let the form go through when submitting a totp code.